my-app/__mocks__/firebase/firestore.js

```
@@ -0,0 +1,67 @@
+  { id: 'mock-id-1', data: () => ({ id: 'mock-id-1', name: 'Mock Item 1' }) },
+  { id: 'mock-id-2', data: () => ({ id: 'mock-id-2', name: 'Mock Item 2' }) },
+]) {
+  return {
+    docs,
+    forEach: function(cb) { docs.forEach(cb); },
+    empty: docs.length === 0,
+    size: docs.length,
+    data: () => ({}),
+    exists: true,
+    get: function() { return this; },
+    then: function(resolve) { resolve(this); },
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)